lib/foreman/observable.rb

```diff
@@ -18,7 +18,7 @@ def event_name_for(hook_name, namespace: DEFAULT_NAMESPACE)
     module_function :event_name_for
 
     def event_payload_for(payload, blk)
-      payload || blk&.call(self)
+      payload || blk&.call(self).to_h
     end
   end
 end
```

Comment:
This change is required because `serialize` from ActiveJob always return regular hash while in subscription API we are using symbols.